sample-compose-fragments/src/main/res/values/dimens.xml

```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0" encoding="utf-8"?><!--
```

Comment:
Is there a way we can make these "more compose" and have them coming from the compose library instead? Then we can keep consistent in our size usage in the components and in the app. (eg. all our existing main app sizes come from the xml component library currently). Maybe similar to the Color Theme class?